### PR TITLE
Resolve SHARE issue

### PR DESCRIPTION
**Pull request by MikeBauerCA**
Signed-off-by: Bauer, Michael <Michael.Bauer2@ca.com>

### CODE REVIEW

**Comment by ChrisBoehmCA** [comment]
Did you know there is a `summary` field that is displayed in the list of groups? This way we could keep the longer description on `zowe provisioning -h`  but have a shorter description on `zowe -h`. But this is also a demonstration so no big deal, haha

**Comment by MikeBauerCA** [comment]
@ChrisBoehmCA thanks for the info. It motivated me to adjust the top-level help so that each zowe group fit on a single line when there was available space.

File: packages/provisioning/src/cli/Provisioning.definition.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ const definition: ICommandDefinition = {
     name: "provisioning",
     aliases: ["pv"],
     type: "group",
+    summary: "Perform z/OSMF provisioning tasks",
```
Comment (zFernand0) [inline packages/provisioning/src/cli/Provisioning.definition.ts]: We need to implement these changes in LTS incremental.
@ChrisBoehmCA, any thoughts?

Comment (ChrisBoehmCA) [reply packages/provisioning/src/cli/Provisioning.definition.ts]: I can open a new PR for LTS incremental.

Comment (zFernand0) [reply packages/provisioning/src/cli/Provisioning.definition.ts]: Thank you :)

**Pull request merged**